openedx_events/learning/data.py

```diff
@@ -0,0 +1,84 @@
+"""
+Data attributes for events within the architecture subdomain `learning`.
+
+These attributes follow the form of attr objects specified in OEP-49 data
+pattern.
+"""
+from datetime import datetime
+from typing import Dict
+
+import attr
+from opaque_keys.edx.keys import CourseKey
+
+
+@attr.s(frozen=True)
+class StudentData:
+    """
+    Attributes defined for Open edX student object.
+    """
+
+    username = attr.ib(type=str)
+    email = attr.ib(type=str)
+    is_active = attr.ib(type=bool, default=True)
+    meta = attr.ib(type=Dict[str, str], factory=dict)
+    name = attr.ib(type=str, factory=str)
+
+
+@attr.s(frozen=True)
+class RegistrationFormData:
+    """
+    Attributes defined for Open edX student object.
+    """
+
+    account_form = attr.ib(type=Dict[str, str], factory=dict)
```

Comment:
Similar to @ormsbee 's note above.  Is this data consistent/important enough that we should parse it before we put it into an event? The flexibility of a generic dict is nice but can lead to breakage because now we have an un-versioned schema inside our nice versioned schema.

Reply:
Thanks for your suggestions! We didn't find a real relevant use case where we could use this Data class, so we decided to remove it. Either way, we could review this later when we start to use the classes.